app/src/main/java/org/mozilla/fenix/components/metrics/MetricsStorage.kt

```diff
@@ -45,23 +45,33 @@ internal class DefaultMetricsStorage(
     /**
      * Checks local state to see whether the [event] should be sent.
      */
+    @Suppress("MagicNumber")
     override suspend fun shouldTrack(event: Event): Boolean =
         withContext(dispatcher) {
-            // The side-effect of storing days of use needs to happen during the first two days after
-            // install, which would normally be skipped by shouldSendGenerally.
+            // The side-effect of storing days of use always needs to happen.
             updateDaysOfUse()
             shouldSendGenerally() && when (event) {
                 Event.GrowthData.SetAsDefault -> {
-                    !settings.setAsDefaultGrowthSent && checkDefaultBrowser()
+                    System.currentTimeMillis().withinFirstMonth() &&
+                        !settings.setAsDefaultGrowthSent &&
+                        checkDefaultBrowser()
                 }
-                Event.GrowthData.FirstAppOpenForDay -> {
-                    settings.resumeGrowthLastSent.hasBeenMoreThanDaySince()
+                Event.GrowthData.FirstWeekSeriesActivity -> {
+                    System.currentTimeMillis().withinFirstMonth() &&
+                        shouldTrackFirstWeekActivity()
                 }
-                Event.GrowthData.FirstUriLoadForDay -> {
-                    settings.uriLoadGrowthLastSent.hasBeenMoreThanDaySince()
+                Event.GrowthData.SerpAdClicked -> {
+                    System.currentTimeMillis().duringSecondDay() && !settings.secondDayAdClickGrowthSent
                 }
-                Event.GrowthData.FirstWeekSeriesActivity -> {
-                    shouldTrackFirstWeekActivity()
+                Event.GrowthData.UriLoaded -> {
+                    if (System.currentTimeMillis().withinFirstDay()) {
+                        val requiredCount = 10
```

Comment:
10 could be a const, something like MAX_NUM_URI_LOADS? That would give the number some context outside the issue, and also remove the need for the suppression

Reply:
Ah, hadn't realized that a class property would make it non-magic. Updated!